Rakefile

```diff
@@ -68,7 +84,20 @@ task :chdir_pkg => [:package_dirs] do
 end
 
 task :deb => [:chdir_pkg, :propsd_source] do
-  sh "fpm --deb-no-default-config-files --deb-recommends nodejs -s dir -t deb -n \"#{name}\" -v #{version} opt/"
+  command = [
+    'fpm',
+    '--deb-no-default-config-files',
+    "--depends \"nodejs = #{target_version}\"",
```

Comment:
I would take a look at this:
https://debian-handbook.info/browse/stable/sect.package-meta-information.html

For example 1.1.1 is not equal to 1.1.1-ubuntu~patch10.303